### PR TITLE
Number_types: Add support for arm and aarch64

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
Define the floating point control word values for arm architectures (for gcc).
## Release Management

* Affected package(s):Number Types

### CODE REVIEW

File: Number_types/include/CGAL/FPU.h [hunk 1]
```diff
@@ -383,6 +383,22 @@ typedef unsigned int FPU_CW_t;
 #define CGAL_FE_TOWARDZERO   _RC_CHOP
 #define CGAL_FE_UPWARD       _RC_UP
 #define CGAL_FE_DOWNWARD     _RC_DOWN
+# elif defined  __arm__
```
Comment (mglisse) [inline Number_types/include/CGAL/FPU.h]: There are several possible FPUs for ARM (plus softfloat), and I think you don't have `__arm__` when using thumb, so I don't know if that is the right test.

Comment (lrineau) [reply Number_types/include/CGAL/FPU.h]: Thanks. I have fixed that in https://github.com/CGAL/cgal/pull/3805/commits/1aee92af751a7021c64082c69dfd2ca6c569c67c. For the moment, I only care about VFP and Neon.

Comment (mglisse) [reply Number_types/include/CGAL/FPU.h]: https://wiki.debian.org/ArmEabiPort suggests `#if (defined(__VFP_FP__) && !defined(__SOFTFP__))` like we already use elsewhere in the file.

Comment (lrineau) [reply Number_types/include/CGAL/FPU.h]: OK. Thanks for the reference. I have committed https://github.com/CGAL/cgal/pull/3805/commits/42cbce9fbf9ad3becb5d1c20ee2eb106b1de7c61.

**Comment by lrineau** [comment]
Was tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-4.14-Ic-160.shtml.

**Pull request merged**